scene/gui/scroll_bar.cpp

```diff
@@ -823,6 +826,8 @@ void ScrollBar::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_drag_slave_input"), &ScrollBar::_drag_slave_input);
 	ClassDB::bind_method(D_METHOD("_drag_slave_exit"), &ScrollBar::_drag_slave_exit);
 
+	ADD_SIGNAL(MethodInfo("scrolling"));
```

Comment:
Though most signals are about something punctual, while this one is (unless you manage to send only one input event) continuous, so it's kind of coherent to me :)